doc/contributing.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+Install development dependencies:
+
+```
+conda env create  # or `mamba env create`
+```
+
+
+## Development install
+
+```
+pip install -e .
+```
+
+
+## Testing the build
+
+```
+rm -rf dist
+python -m build
+pip install dist/*.whl  # or `dist/*.tar.gz`
```

Comment:
What is this `or` clause for?

Reply:
It's just a comment to point out the two different distributions that could be installed and tested locally. If the wheel works, the source dist should definitely work, so maybe not worth calling this out.